ingestion/src/metadata/cmd.py

```diff
@@ -110,20 +113,20 @@ def test(config: str) -> None:
 def profile(config: str) -> None:
     """Main command for profiling Table sources into Metadata"""
     config_file = pathlib.Path(config)
-    workflow_config = load_config_file(config_file)
-
+    workflow_config_dict = None
```

Comment:
Do we need it to assign it `None` or could just go for the assignment in `try:` directly?

Reply:
I am assigning `None` since `workflow_config_dict` is used in the exception catch, and the method `load_config_file` could throw an exception.

```python
workflow_config_dict = None
try:
    workflow_config_dict = load_config_file(config_file)
    logger.debug(f"Using config: {workflow_config_dict}")
    workflow = ProfilerWorkflow.create(workflow_config_dict)
except Exception as exc:
    logger.debug(traceback.format_exc())
    print_init_error(exc, workflow_config_dict, WorkflowType.profile) # <-- here
    sys.exit(1)
```

Reply:
Make sense. Apologies for missing that 😅